addons/mail/static/src/new/rtc/call_invite_request_popup.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.call_invite_request_popup" owl="1">
+        <div class="o_CallInviteRequestPopup d-flex flex-column m-2 p-5 border border-dark rounded-1 bg-900" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
note (ok to change in follow up PR) this doesn't follow new class name guideline